toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -529,7 +529,7 @@ python3-libcap-ng-0.8.2-2.cm2.x86_64.rpm
 python3-libs-3.9.19-6.cm2.x86_64.rpm
 python3-libxml2-2.10.4-4.cm2.x86_64.rpm
 python3-lxml-4.9.1-1.cm2.x86_64.rpm
-python3-magic-5.40-2.cm2.noarch.rpm
+python3-magic-5.40-3.cm2.noarch.rpm
```

Comment:
why python3-magic?  Is your branch out of date?

Reply:
Python3-magic is actually shipped by file.
File looks at magic numbers and the upstream decided to probably name it this way.
This is a subpacakge of file